.github/DISCUSSION_TEMPLATE/rfc.md

```diff
@@ -0,0 +1,95 @@
+
+
+Keep in mind RFCs are organic documents, it is reasonable and even expected that some portions of it need to be revisited (i.e. adjusting the definition of the Problem Space after some initial investigation into Approaches).
+
+
+Generally, involve the team as early and often as possible, but there are specific inflection points noted where this is required.
+--->
+
+
+| Author(s)    | Created        | Project |
+|--------------|----------------|---------|
+| Your name(s) | <Today's Date> |         |
+
+
+# Goals
+
+
+<!--- Describe the outcome(s) you’re looking to achieve from this RFC. This should be something that is then actionable. Be specific but keep it to about a paragraph. If it takes more to describe what is trying to be done, then it probably needs to be more than one RFC. Some categories that this could fall under:
+   * Making a technology choice that will have medium-to-high impact on the team and/or project(s).
+   * Making a decision about the architectural approach to part of a project before implementing it.
+   * Determining the best approach to improve an existing codebase that is proving complex / difficult to maintain.
+--->
+
+# Prior Art
+
+
+<!--- Note any existing implementations we may have:
+   * For that implementation, did we try anything different from other existing implementations?
+   * What worked in this implementation?
+   * What was less than ideal in this implementation?
+   * Did we make any tradeoffs in our approach that we don’t want to repeat or were specific to that project?
+Link any RFCs that have addressed/looked at this in the past. --->
+
+
+# Problem Space
+
+<!---
+1. Describe the problem space for the problem you’re trying to solve, this should be to the level of detail required for a new engineer to read this and comprehend the specifics of the problem and any nuances that may be present.
+2. At this point, the RFC should be shared with the team to collect comprehensive knowledge on the entire team’s experience with this problem space. The team should comment on this document in regards to that and the author(s) should incorporate that information into the definition of the problem space.
+3. You may choose to defer the Approaches portion of this RFC until after the Problem Space has been satisfactorily defined. However, there may be some obvious solutions from the start that are beneficial to define earlier in the process of developing your RFC.
+--->
+
+# Approaches
+
+
+
+<!---
+1. Taking the Goals and Problem Space, begin to investigate solutions to the Problem Space. This may take the form of (but not limited to):
+   * Architectural design
+   * Data design
+   * Infrastructure requirements
+   * R&D into existing tools/solutions
+2. The goal here is to consider as many options as possible with a practical amount of time and effort.
+3. An iterative approach is best here:
+   * Start with outlining broadly the approaches that could be used. 
```

Comment:
I think simply describing different approaches means you are leaving the decisions for someone else to make.  Author should be able to propose the approach she thinks would work best .

Reply:
e.g. https://docs.google.com/document/d/1Oz_7FhaWxdFUDEzKCC5Cy58t57C4znmC_Qr80BORy1U/edit#heading=h.7r3q85c5fjk

proposal  comes first and then "abandoned ideas" are optional